Developer-Guide.md

```diff
@@ -517,12 +517,14 @@ $ sudo sed -i '$a Requires=kata-debug.service' ${ROOTFS_DIR}/lib/systemd/system/
 ### Build the debug image
 
 Follow the instructions in the [Build a rootfs image](#build-a-rootfs-image)
-section.
+section when using rootfs, or for initrd, complete the steps in the [Build an initrd image](#build-an-initrd-image) section.
 
 ### Configure runtime for custom debug image
 
 Install the image:
 
+**Note**: When using an initrd image, use the image name `kata-containers.img` below, in place of the rootfs image name `kata-containers.img`.
```

Comment:
nit: the first `kata-containers.img` should be `kata-containers-initrd.img`

Reply:
Thank you @chavafg. Will change it.

Reply:
thanks @Amulyam24 but I still see this line needs to be changed, you have:

>**Note**: When using an initrd image, use the image name `kata-containers.img` below, in place of the rootfs image name `kata-containers-initrd.img`.

And should be:

> **Note**: When using an initrd image, use the image name `kata-containers-initrd.img` below, in place of the rootfs image name `kata-containers.img`.

am I correct @grahamwhaley ? Or I'm getting confused because of the wording?  😄

Reply:
I think you are correct @chavafg . I would probably write it as:

> **Note:** When using an initrd image, replace the below rootfs image name `kata-containers.img` with the initrd image name `kata-containers-initrd.img`.

Reply:
Yes. That has to be changed. Thanks @chavafg